### ISSUE TITLE
trouble in dev env with css assets

### ISSUE THREAD

**Issue by eliotik** [braincrafted/bootstrap-bundle#90]
Hello, i have setup all configs as in tutorial but when i run dev env as
http://localhost/app_dev.php
no css rules are loaded, but in html code i can see url to generated file:
<link href="/app_dev.php/css/9f756df.css" type="text/css" rel="stylesheet" />
and the content of that file is result of merging 2 .less files:
bootstrap.less
responsive.less

> /*!
> - Bootstrap v2.3.2
>   *
> - Copyright 2013 Twitter, Inc
> - Licensed under the Apache License v2.0
> - http://www.apache.org/licenses/LICENSE-2.0
>   *
> - Designed and built with all the love in the world by @mdo and @fat.
>   _/// Core variables and mixins @import "variables.less";// Modify this for custom > colors,font-sizes,etc @import "mixins.less";// CSS Reset @import "reset.less";// > Grid system and page structure @import "scaffolding.less";@import
>   "grid.less";@import "layouts.less";// Base CSS @import "type.less";@import
>   "code.less";@import "forms.less";@import "tables.less";// Components:common > @import "sprites.less";@import "dropdowns.less";@import "wells.less";@import > "component-animations.less";@import "close.less";// Components:Buttons &
>   Alerts @import "buttons.less";@import "button-groups.less";@import 
>   "alerts.less";// Note:alerts share common CSS with buttons and thus have styles > in buttons.less // Components:Nav @import "navs.less";@import 
>   "navbar.less";@import "breadcrumbs.less";@import "pagination.less";@import 
>   "pager.less";// Components:Popovers @import "modals.less";@import 
>   "tooltip.less";@import "popovers.less";// Components:Misc @import 
>   "thumbnails.less";@import "media.less";@import "labels-badges.less";@import 
>   "progress-bars.less";@import "accordion.less";@import "carousel.less";@import > "hero-unit.less";// Utility classes @import "utilities.less";// Has to be last to
>   override when necessary
>   /_!
> - Bootstrap Responsive v2.3.2
>   *
> - Copyright 2013 Twitter, Inc
> - Licensed under the Apache License v2.0
> - http://www.apache.org/licenses/LICENSE-2.0
>   *
> - Designed and built with all the love in the world by @mdo and @fat.
>   */// Responsive.less // For phone and tablet devices // 
>   ------------------------------------------------------------- // REPEAT VARIABLES & MIXINS // > ------------------------- // Required since we compile the responsive stuff separately 
>   @import "variables.less";// Modify this for custom colors,font-sizes,etc @import > "mixins.less";// RESPONSIVE CLASSES // ------------------ @import "responsive-
>   utilities.less";// MEDIA QUERIES // ------------------ // Large desktops @import 
>   "responsive-1200px-min.less";// Tablets to regular desktops @import 
>   "responsive- 768px-979px.less";// Phones to portrait tablets and narrow desktops > @import "responsive-767px-max.less";// RESPONSIVE NAVBAR // ------------------ > // From 979px and below,show a button to toggle navbar contents @import 
>   "responsive-navbar.less";

not the result of less. But when i use prod env with http://localhost/app.php or "http://localhost/" all work good.

i have symfony 2.3.4
in require:

> "braincrafted/bootstrap-bundle": "dev-master",
> "jquery/jquery": "1.9._",
> "leafo/lessphp": "v0.4.0",
> "twbs/bootstrap": "2.3._",

$ appconsole assetic:dump

> Dumping all dev assets.
> Debug mode is on.
> 15:11:29 [file+] /var/www/app/../web/css/bootstrap.css
> 15:11:44 [file+] /var/www/app/../web/css/bootstrap_bootstrap_1.css
> 15:11:55 [file+] /var/www/app/../web/css/bootstrap_responsive_2.css
> 15:11:59 [file+] /var/www/app/../web/js/bootstrap.js
> 15:11:59 [file+] /var/www/app/../web/js/bootstrap_bootstrap-transition_1.js
> 15:11:59 [file+] /var/www/app/../web/js/bootstrap_bootstrap-alert_2.js
> 15:11:59 [file+] /var/www/app/../web/js/bootstrap_bootstrap-button_3.js
> 15:11:59 [file+] /var/www/app/../web/js/bootstrap_bootstrap-carousel_4.js
> 15:11:59 [file+] /var/www/app/../web/js/bootstrap_bootstrap-collapse_5.js
> 15:11:59 [file+] /var/www/app/../web/js/bootstrap_bootstrap-dropdown_6.js
> 15:11:59 [file+] /var/www/app/../web/js/bootstrap_bootstrap-modal_7.js
> 15:11:59 [file+] /var/www/app/../web/js/bootstrap_bootstrap-tooltip_8.js
> 15:11:59 [file+] /var/www/app/../web/js/bootstrap_bootstrap-popover_9.js
> 15:11:59 [file+] /var/www/app/../web/js/bootstrap_bootstrap-scrollspy_10.js
> 15:11:59 [file+] /var/www/app/../web/js/bootstrap_bootstrap-tab_11.js
> 15:11:59 [file+] /var/www/app/../web/js/bootstrap_bootstrap-typeahead_12.js
> 15:11:59 [file+] /var/www/app/../web/js/bootstrap_bootstrap-affix_13.js
> 15:11:59 [file+] /var/www/app/../web/js/jquery.js
> 15:11:59 [file+] /var/www/app/../web/js/jquery_jquery-1.9.1_1.js
> 15:11:59 [file+] /var/www/app/../web/css/9f756df.css
> 15:12:20 [file+] /var/www/app/../web/css/9f756df_part_1.css
> 15:12:41 [file+] /var/www/app/../web/js/eb49830.js
> 15:13:04 [file+] /var/www/app/../web/js/eb49830_part_1.js
> 15:13:26 [file+] /var/www/app/../web/js/cd44034.js
> 15:13:26 [file+] /var/www/app/../web/js/cd44034_part_1.js

$ appconsole assetic:dump --env=prod --no-debug

> Dumping all prod assets.
> Debug mode is off.
> 15:14:05 [file+] /var/www/app/../web/css/bootstrap.css
> 15:14:20 [file+] /var/www/app/../web/js/bootstrap.js
> 15:14:20 [file+] /var/www/app/../web/js/jquery.js
> 15:14:20 [file+] /var/www/app/../web/css/9f756df.css
> 15:14:41 [file+] /var/www/app/../web/js/eb49830.js
> 15:15:03 [file+] /var/www/app/../web/js/cd44034.js

file 9f756df.css has normal result of less and yuicompressor, all css rules.
and if i directly set path to that file, page looks good.

config next:

``` yaml
bc_bootstrap:
    less_filter: lessphp
assetic:
    debug:          %kernel.debug%
    use_controller: false
    bundles:        ['AcmeTestBundle']
    #java: /usr/bin/java
    filters:
        lessphp:
            file: %kernel.root_dir%/../vendor/leafo/lessphp/lessc.inc.php
            apply_to: "\.less$"
        less:
            node: /usr/local/bin/node
            node_paths: [/usr/local/node/node_modules]
            apply_to: "\.less$"
        cssrewrite: ~
        #closure:
        #    jar: %kernel.root_dir%/Resources/java/compiler.jar
        yui_js:
            jar: %kernel.root_dir%/Resources/java/yuicompressor-2.4.8.jar
        yui_css:
            jar: %kernel.root_dir%/Resources/java/yuicompressor-2.4.8.jar
    assets:
        jquery:
            inputs:
                  - %kernel.root_dir%/../vendor/jquery/jquery/jquery-1.9.1.js
            output: js/jquery.js
        bootstrap_css:
            inputs:
                - %kernel.root_dir%/../vendor/twbs/bootstrap/less/bootstrap.less
                - %kernel.root_dir%/../vendor/twbs/bootstrap/less/responsive.less
            filters:
                - lessphp
                - cssrewrite
            output: css/bootstrap.css
        bootstrap_js:
            inputs:
                - %kernel.root_dir%/../vendor/twbs/bootstrap/js/bootstrap-transition.js
                - %kernel.root_dir%/../vendor/twbs/bootstrap/js/bootstrap-alert.js
                - %kernel.root_dir%/../vendor/twbs/bootstrap/js/bootstrap-button.js
                - %kernel.root_dir%/../vendor/twbs/bootstrap/js/bootstrap-carousel.js
                - %kernel.root_dir%/../vendor/twbs/bootstrap/js/bootstrap-collapse.js
                - %kernel.root_dir%/../vendor/twbs/bootstrap/js/bootstrap-dropdown.js
                - %kernel.root_dir%/../vendor/twbs/bootstrap/js/bootstrap-modal.js
                - %kernel.root_dir%/../vendor/twbs/bootstrap/js/bootstrap-tooltip.js
                - %kernel.root_dir%/../vendor/twbs/bootstrap/js/bootstrap-popover.js
                - %kernel.root_dir%/../vendor/twbs/bootstrap/js/bootstrap-scrollspy.js
                - %kernel.root_dir%/../vendor/twbs/bootstrap/js/bootstrap-tab.js
                - %kernel.root_dir%/../vendor/twbs/bootstrap/js/bootstrap-typeahead.js
                - %kernel.root_dir%/../vendor/twbs/bootstrap/js/bootstrap-affix.js
            output: js/bootstrap.js
```

content of twig template:

``` html
{% block content %}
    <div class="btn-group btn-group-vertical">
        <button class="btn" type="button"><i class="icon-align-left"></i></button>
        <button class="btn" type="button"><i class="icon-align-center"></i></button>
        <button class="btn" type="button"><i class="icon-align-right"></i></button>
        <button class="btn" type="button"><i class="icon-align-justify"></i></button>
    </div>

    {% block mainSectionContent %}{% endblock %}

{% endblock %}

{% block stylesheets %}
    {% stylesheets '@bootstrap_css' combine=true filter='yui_css' %}
    <link href="{{ asset_url }}" type="text/css" rel="stylesheet" />
    {% endstylesheets %}
{% endblock %}

{% block javascripts %}
    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
    {% javascripts '@bootstrap_js' combine=true filter='yui_js' %}
    <script src="{{ asset_url }}"></script>
    {% endjavascripts %}
{% endblock %}

{% block footerjavascripts %}
    <script>window.jQuery || document.write('{% javascripts '@jquery' %}<script src="{{ asset_url }}"><\/script>{% endjavascripts %}');</script>
{% endblock %}
```

anyone could help me to find what am i missing?
thank you.
